CMP1903_A1_2324/Testing.cs

```diff
@@ -1,19 +1,42 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CMP1903_A1_2324
 {
-    internal class Testing
+    public class Testing
     {
         /*
          * This class should test the Game and the Die class.
          * Create a Game object, call the methods and compare their output to expected output.
          * Create a Die object and call its method.
          * Use debug.assert() to make the comparisons and tests.
          */
+        public static void Test()
+       {
+           Game game = new Game();
```

Comment:
Would recommend commenting this line and the below line to make sure the user knows what this does as they are very important.